lib/jekyll-feed/feed.xml

```diff
@@ -86,6 +86,7 @@
           {% assign post_image = post_image | absolute_url %}
         {% endunless %}
         <media:thumbnail xmlns:media="http://search.yahoo.com/mrss/" url="{{ post_image | xml_escape }}" />
+        <media:content url="{{ post_image | xml_escape }}" medium="image">
```

Comment:
Looks like you're missing a proper closing tag here, which we can get by just adding a `/` before the `>`

```suggestion
        <media:content url="{{ post_image | xml_escape }}" medium="image" />
```

Reply:
Thanks, @mattr-!  Sorry, I just saw this but it looks like someone else already took care of it.